core/file_walker.go

```diff
@@ -60,9 +60,12 @@ func GetSubTree(path string, parent *File, readDir ReadDir, ignoredFolders map[s
 			files = append(files, file)
 		}
 	}
-	ret.Name = name
 	if parent != nil {
+		ret.Name = name
 		ret.Parent = parent
+	} else {
+		// Root dir
+		ret.Name = dirName + name
```

Comment:
I'm not sure I understand your comment.
what about putting a line after this: https://github.com/viktomas/godu/blob/master/godu.go#L25

where we would say `tree.name = root` ?